src/app/[locale]/(unauth)/docs/DocsContent.tsx

```diff
@@ -68,191 +76,254 @@ export default function DocsContent() {
+          <li><strong>pdf.started</strong></li>
+          <li><strong>pdf.generated</strong></li>
+          <li><strong>pdf.failed</strong></li>
         </ul>
 
-        <h3>
-          Example&nbsp;
-          <strong>curl</strong>
-          {' '}
-          to preview dev version:
-        </h3>
+        <h3>Webhook Example Payloads</h3>
 
-        <pre className="overflow-x-auto rounded-lg bg-gray-800 p-4 dark:bg-gray-800">
-          <code>
-            {`curl --location https://api.templify.cloud/convert/YOUR_TEMPLATE_ID_HERE?devMode=true' \\
---header 'client_id: USER_ID_HERE' \\
---header 'client_secret: CLIENT_SECRET_HERE' \\
---header 'Content-Type: application/json' \\
---header 'Cookie: NEXT_LOCALE=en' \\
---data '{
-  "templateData": {
-         "name": "John Doe",
-         "invoice_number": "INV-1001",
-         "items": [
-           { "description": "Item 1", "price": 20 },
-           { "description": "Item 2", "price": 30 }
-         ]
-       }
-     }'`}
-          </code>
+        <h4>📤 pdf.started</h4>
+        <pre className="overflow-x-auto rounded bg-gray-800 p-4">
+          {`{
+  "id": "evt_84h1N3P2",
+  "type": "pdf.started",
+  "created_at": "2025-11-02T10:12:00Z",
+  "attempt": 1,
+  "meta": {
+    "input_data": { "company_name": "ABC Corp" }
+  }
+}`}
+        </pre>
+
+        <h4>📤 pdf.generated</h4>
+        <pre className="overflow-x-auto rounded bg-gray-800 p-4">
+          {`{
+  "id": "evt_84h1N3P2",
+  "type": "pdf.generated",
+  "created_at": "2025-11-02T10:12:00Z",
+  "data": {
+    "download_url": "https://cdn.templify.cloud/renders/xyz.pdf",
+    "render_ms": 12450,
+    "expires_at": "2025-11-03T10:12:00Z"
+  }
+}`}
         </pre>
 
-        <h3>✅ Production PDF Generation (Default Behavior)</h3>
+        <h4>📤 pdf.failed</h4>
+        <pre className="overflow-x-auto rounded bg-gray-800 p-4">
+          {`{
+  "id": "evt_84h1N3P3",
+  "type": "pdf.failed",
+  "data": {
+    "error": {
+      "code": "TEMPLATE_NOT_FOUND",
+      "message": "Template ID tmpl_999 does not exist.",
+      "details": null
+    },
+    "render_ms": 350
+  },
+  "meta": {
+    "template_id": "tmpl_999",
+    "job_id": "job_8r9sm1a3",
+    "input_data": { "company_name": "ABC Corp" }
+  }
+}`}
+        </pre>
+      </div>
+
+      {/* SIGNATURE VALIDATION */}
+      <div id="signature">
+        <h2>🔏 Webhook Signature Verification</h2>
+
+        <p>Every webhook request includes the header:</p>
+        <pre className="overflow-x-auto rounded bg-gray-800 p-4">{`x-templify-signature: sha256=<signature>`}</pre>
+
+        <p>You should validate the signature on your server:</p>
+
+        <pre className="overflow-x-auto rounded bg-gray-800 p-4">
+          {`const crypto = require("crypto");
+
+app.post("/webhooks/templify", (req, res) => {
+  const signature = req.headers["x-templify-signature"];
+  const payload = JSON.stringify(req.body);
+  const secret = process.env.TEMPLIFY_WEBHOOK_SECRET;
```

Comment:
Add a documentation note on where will the user be able to get this templify webhook secret from?
In our case it will be generated when the user adds a webhook url.
Also, add a settings/webhooks page link here